### ISSUE TITLE
Source control shows empty original file in side-by-side comparison when project is located on file share

### ISSUE THREAD

**Issue by rdenneman** [microsoft/vscode#142603]
Issue Type: <b>Bug</b>

1. Clone a git repository on a remote file share
2. Open folder \\\remote_system\project_folder
3. Edit one of the files
4. Open source control pane
5. Select modified file

Expected result: Side-by-side comparison opens showing changes

Actual result: Side-by-side comparison opens but the original file is empty and all lines in the file are marked as new. Running git diff in terminal on the file does show the modified lines.

VS Code version: Code 1.64.1 (d6ee99e4c045a6716e5c653d7da8e9ae6f5a8b03, 2022-02-07T17:16:22.147Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (4 x 2808)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.86GB (18.00GB free)|
|Process Argv|--crash-reporter-id 32eba180-7fe3-4179-bbbd-4c1a47ffb697|
|Screen Reader|no|
|VM|60%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
qlik|Gim|0.5.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627cf:30244335
pythonvspyl392cf:30425750
pythontb:30283811
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30433059
pythonvs932:30410667
wslgetstartedc:30433508
vscop453:30404998
vsrem710:30416614

```

</details>

**Comment by yongnuo** [comment]
What is the status of this bug? I'm experiencing the same problem with version 1.69.1

Version: 1.69.1 (user setup)
Commit: b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a
Date: 2022-07-12T08:21:24.514Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.22000

**Comment by rdenneman** [comment]
I'm using the Remote SSH plugin as well. Since I reported this bug, more bugs relating to working on samba shares have popped up.

**Comment by lszomoru** [comment]
This issue looks similar to https://github.com/microsoft/vscode/issues/142842, which has been fixed earlier this week, and the fix is currently available in the Insiders release, and it will be included in the next Stable release. I will go ahead and close this issue as duplicate of https://github.com/microsoft/vscode/issues/142842.

**Comment by rdenneman** [comment]
Is indeed fixed in insiders v1.70.0
